grizli/model.py

```diff
@@ -3875,17 +3882,18 @@ def __init__(self, flt=None, beam=None, conf=None,
         # self.min_sens = min_sens
         # self.mask_resid = mask_resid
 
-        self._parse_from_data(**self._parse_params)
+        self._parse_from_data(isJWST=isJWST, **self._parse_params)
 
     def _parse_from_data(self, contam_sn_mask=[10, 3], min_mask=0.01,
-                         seg_ids=None, min_sens=0.08, mask_resid=True):
+                         seg_ids=None, min_sens=0.08, mask_resid=True, isJWST=False):
         """
         See parameter description for `~grizli.model.BeamCutout`.
         """
         # bad pixels or problems with uncertainties
-        self.mask = ((self.grism.data['DQ'] > 0) |
-                     (self.grism.data['ERR'] == 0) |
-                     (self.grism.data['SCI'] == 0))
+        self.mask = ((self.grism.data['DQ'] > 0) | 
+                    (self.grism.data['ERR'] == 0) |
```

Comment:
i think add back a space before `(` after the first line so that they don't show up as a change?

Reply:
Fixed here (I think?): 43c6ccd08f26c47b8448ef796eaddc0419d47be4